src/routes/users.ts

```diff
@@ -26,5 +29,31 @@ export const users = (): Router => {
       return res.status(500).json({ message: err.message });
     }
   });
+
+  type InfoType = {
+    body: {
+      name: string;
+    };
+  };
+
+  router.put('/me/update-info', accessTokenParser, loginRequired, validator.updateUserInfo, apiValidatorMiddleware, async (req: WebevRequest & InfoType, res: Response) => {
```

Comment:
put の中に 更新の意味合いが含まれているので endpoint に update は含めない方が良いですね
```suggestion
  router.put('/me/info', accessTokenParser, loginRequired, validator.updateUserInfo, apiValidatorMiddleware, async (req: WebevRequest & InfoType, res: Response) => {
```